mypy/checker.py

```diff
@@ -1647,6 +1647,24 @@ def visit_class_def(self, defn: ClassDef) -> None:
                 with self.scope.push_class(defn.info):
                     self.accept(defn.defs)
             self.binder = old_binder
+            for base in typ.mro[1:]:
```

Comment:
Could you please add a short comment explaining that the class itself is skipped on purpose because this is how it works at runtime? Also explain why `object` is skipped below.